planning/behavior_path_start_planner_module/README.md

```diff
@@ -2,24 +2,73 @@
 
 ## Purpose / Role
 
-The Start Planner module is designed to generate a path from the current ego position to the driving lane, avoiding static obstacles and stopping in response to dynamic obstacles when a collision is detected.
+This module generates and plans a path for safely merging from the shoulder lane into the travel lane.
 
-Use cases include:
+Specifically, it includes the following features:
 
-- pull out from the side of the road lane to centerline.
+- The ability to automatically start from the shoulder lane.
+- When parked vehicles are present on the shoulder lane, the module generates a path that allows for starting with a gap of a specified margin.
+- If a collision with other traffic participants is detected while traveling on the generated path, it will stop as much as possible.
```

Comment:
not just shoulder lane

Reply:
sorry fixed in https://github.com/autowarefoundation/autoware.universe/pull/6264/commits/29d143817702013686e86cc031616fdc219a83ff